src/OhadSoft.AzureLetsEncrypt.Renewal/Management/SharedRenewalParameters.cs

```diff
@@ -54,10 +53,5 @@ public sealed class SharedRenewalParameters
         public Uri AzureTokenAudience { get; }
         public Uri AzureManagementEndpoint { get; }
         public string AzureDefaultWebsiteDomainName { get; }
-
-        public override string ToString()
```

Comment:
are you sure this is not printed out anywhere?

Reply:
they (ToString() on this class as well as  AzureEnvironmentParams) was only called indirectly when RenewalParameters .ToString() was called as part of the sendgrid notification

Reply:
resolving this for now but I am happy to revisit this if ive overlooked something